docs/gradle-plugin.md

```diff
@@ -0,0 +1,138 @@
+(gradle-plugin-page)=
+
+# Using the Gradle plugin
+
+The [Gradle plugin for Nextflow plugins](https://github.com/nextflow-io/nextflow-plugin-gradle) simplifies plugin development by configuring default dependencies needed for Nextflow integration and incorporates custom Gradle tasks that streamline building, testing, and publishing Nextflow plugins. This guide describes how to use the Gradle plugin for plugin development.
```

Comment:
We are going to use  [this project](https://github.com/nextflow-io/nf-plugin-template/) as "official" template project for plugin

Reply:
More descriptions and links pointing to the template have been added throughout the docs.